### PR TITLE
[network] Marking CRDs to be backed up by d8 cli

### PR DESCRIPTION
**Pull request by apolovov**
## Description
Marking network module's CRDs to be backed up by d8 cli.
Also, fixed `IngressIstioController` CRD docs rendering.

## Why do we need it, and what problem does it solve?
The related to CRDs objects will be backed up by d8 cli.

## What is the expected result?
```
kubectl get crd -l backup.deckhouse.io/cluster-config=true
```
must contain
```
egressgatewaypolicies.network.deckhouse.io
egressgateways.network.deckhouse.io
iprulesets.network.deckhouse.io
routingtables.network.deckhouse.io
istiofederations.deckhouse.io
istiomulticlusters.deckhouse.io
ingressistiocontrollers.deckhouse.io
keepalivedinstances.deckhouse.io
ciliumclusterwidenetworkpolicies.cilium.io
```

## Checklist
- [ ] The code is covered by unit tests.
- [x] e2e tests passed.
- [x] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
---
section: tools
type: chore
summary: Network module's CRDs marked to be backed up by d8 cli.
---
section: istio
type: fix
summary: Fixed `IngressIstioController` CRD docs rendering.
```

### CODE REVIEW

File: modules/110-istio/_crds/istio/1.19/crd-all.gen.yaml [hunk 1]
```diff
@@ -3505,6 +3507,7 @@ metadata:
     chart: istio
     heritage: Tiller
     release: istio
+    backup.deckhouse.io/cluster-config: "true"
```
Comment (apolovov) [inline modules/110-istio/_crds/istio/1.19/crd-all.gen.yaml:3510]:
```suggestion
```

File: modules/110-istio/_crds/istio/1.19/crd-all.gen.yaml [hunk 2]
```diff
@@ -3571,6 +3574,7 @@ metadata:
...
```
Comment (apolovov) [inline modules/110-istio/_crds/istio/1.19/crd-all.gen.yaml:3577]:
```suggestion
```

File: modules/110-istio/_crds/istio/1.19/crd-all.gen.yaml [hunk 3]
```diff
@@ -3846,6 +3850,7 @@ metadata:
...
```
Comment (apolovov) [inline modules/110-istio/_crds/istio/1.19/crd-all.gen.yaml:3853]:
```suggestion
```

File: modules/110-istio/_crds/istio/1.19/crd-all.gen.yaml [hunk 4]
```diff
@@ -4219,6 +4224,7 @@ metadata:
...
```
Comment (apolovov) [inline modules/110-istio/_crds/istio/1.19/crd-all.gen.yaml:4227]:
```suggestion
```

File: modules/110-istio/_crds/istio/1.19/crd-all.gen.yaml [hunk 5]
```diff
@@ -5892,6 +5898,7 @@ metadata:
...
```
Comment (apolovov) [inline modules/110-istio/_crds/istio/1.19/crd-all.gen.yaml:5901]:
```suggestion
```

File: modules/110-istio/_crds/istio/1.19/crd-all.gen.yaml [hunk 6]
```diff
@@ -6021,6 +6028,7 @@ metadata:
...
```
Comment (apolovov) [inline modules/110-istio/_crds/istio/1.19/crd-all.gen.yaml:6031]:
```suggestion
```

File: modules/110-istio/_crds/istio/1.19/crd-all.gen.yaml [hunk 7]
```diff
@@ -6349,6 +6357,7 @@ metadata:
     heritage: Tiller
     istio: security
     release: istio
+    backup.deckhouse.io/cluster-config: "true"
```
Comment (apolovov) [inline modules/110-istio/_crds/istio/1.19/crd-all.gen.yaml:6360]:
```suggestion
```

File: modules/110-istio/_crds/istio/1.19/crd-all.gen.yaml [hunk 8]
```diff
@@ -6739,6 +6748,7 @@ metadata:
...
```
Comment (apolovov) [inline modules/110-istio/_crds/istio/1.19/crd-all.gen.yaml:6751]:
```suggestion
```

File: modules/110-istio/_crds/istio/1.19/crd-all.gen.yaml [hunk 9]
```diff
@@ -6832,6 +6842,7 @@ metadata:
...
```
Comment (apolovov) [inline modules/110-istio/_crds/istio/1.19/crd-all.gen.yaml:6845]:
```suggestion
```

File: modules/110-istio/_crds/istio/1.19/crd-all.gen.yaml [hunk 10]
```diff
@@ -7028,6 +7039,7 @@ metadata:
     heritage: Tiller
     istio: telemetry
     release: istio
+    backup.deckhouse.io/cluster-config: "true"
```
Comment (apolovov) [inline modules/110-istio/_crds/istio/1.19/crd-all.gen.yaml:7042]:
```suggestion
```

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @apolovov. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @apolovov. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by apolovov** [comment]
/e2e/abort yandex-cloud;WithoutNAT;containerd;1.30 11786683587 11786683587-1-con-1-30 /sys/deckhouse-oss/install:pr10581 redos@89.169.132.212

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @apolovov. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @apolovov. I've started the workflow for label 'e2e/run/static'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @apolovov. I've started the workflow for label 'deploy/web/stage'...

**Review by nabokihms — APPROVED** [review]

**Pull request merged**